### PR TITLE
shacl-compact-syntax sec A. IANA Considerations

### PR DESCRIPTION
**Pull request by VladimirAlexiev**

### CODE REVIEW

**Comment by VladimirAlexiev** [comment]
@HolgerKnublauch

File: shacl-compact-syntax/index.html [hunk 1]
```diff
@@ -716,12 +723,76 @@ <h2>Grammar and Production Rules</h2>
...
+        </dd>
+        <dt>Additional information:</dt>
+        <dd>
+          <dl>
+            <dt>Magic number(s):</dt>
+            SHACLC documents will likely have the words <code>PREFIX</code> or <code>BASE</code> (case sensitive) near the beginning of the document.
+            However, the same words may appear in Turtle and SPARQL documents.
+            <dd></dd>
+            <dt>File extension(s):</dt>
+            <dd>.shaclc</dd>
```
Comment (VladimirAlexiev) [inline shacl-compact-syntax/index.html:771]: @HolgerKnublauch please add `.shc` as per Andy's request. I've added it in my IANA submission (IANA tracking #1172417)

**Comment by HolgerKnublauch** [comment]
Something has changed in the process here. Even I can no longer push to master without reviews. So I have created this branch https://github.com/w3c/shacl/compare/add-shc?expand=1 and was trying to add you to the reviewers, but I don't know which user name and VladimirAlexiev cannot be added. Can you review this manually somehow?

**Comment by VladimirAlexiev** [comment]
@HolgerKnublauch When I click your link, I get to a page where I can create a new PR:
image

but have no buttons to accept your change. 
I've been member of the w3c org but even when I change "perspective" to that org, I still don't see buttons to accept the change.

So instead, I made an equivalent PR: https://github.com/w3c/shacl/pull/5, are you able to accept that?

BTW, where is the source of http://datashapes.org/constraints.html, I'd like to add `dash:valueIn`

**Pull request merged**